lib/turbolinks/redirection.rb

```diff
@@ -6,11 +6,13 @@ module Redirection
     def redirect_to(url = {}, response_status = {})
       turbolinks, options = _extract_turbolinks_options!(response_status)
 
-      super(url, response_status)
+      value = super(url, response_status)
```

Comment:
`value` is a badly generic name; I'm not sure what type of object is returned by a call to `redirect_to`